x-pack/test/load/runner.ts

```diff
@@ -9,27 +9,50 @@ import { resolve } from 'path';
 import { REPO_ROOT } from '@kbn/utils';
 import { FtrProviderContext } from './../functional/ftr_provider_context';
 
+/**
+ *
+ * GatlingTestRunner is used to run load simulation against local Kibana instance
+ *
+ * Use GATLING_SIMULATIONS to pass comma-separated class names
+ * Use GATLING_PROJECT_PATH to override path to 'kibana-load-testing' project
+ */
 export async function GatlingTestRunner({ getService }: FtrProviderContext) {
   const log = getService('log');
-  const gatlingProjectRootPath = resolve(REPO_ROOT, '../kibana-load-testing');
 
   await withProcRunner(log, async (procs) => {
-    await procs.run('gatling', {
+    const simulationPackage = 'org.kibanaLoadTest.simulation';
+    const gatlingProjectRootPath: string =
+      process.env.GATLING_PROJECT_PATH || resolve(REPO_ROOT, '../kibana-load-testing');
+    const simulations: string = process.env.GATLING_SIMULATIONS || 'DemoJourney';
+    await procs.run('mvn: clean compile', {
       cmd: 'mvn',
       args: [
-        'clean',
-        '-q',
+        '-Dmaven.wagon.http.retryHandler.count=3',
         '-Dmaven.test.failure.ignore=true',
-        'compile',
-        'gatling:test',
         '-q',
-        '-Dgatling.simulationClass=org.kibanaLoadTest.simulation.DemoJourney',
+        'clean',
+        'compile',
       ],
       cwd: gatlingProjectRootPath,
       env: {
         ...process.env,
       },
       wait: true,
     });
+    for (const simulationClass of simulations.split(',').filter((i) => i.length > 0)) {
```

Comment:
nit: This could be a named fn:
```js
const dropEmptyLines = x => x.split(',').filter(i => i.length > 0);
for (const simulationClass of dropEmptyLines(simulations)) {
...
```
Not a big deal at all, just a "nit pick" lol.